src/opencl/image_function_opencl.cpp

```diff
@@ -292,6 +293,25 @@ namespace
             }
         }
 
+        __kernel void setPixelOpenCL( __global const uchar * data, uint offset, uint rowSize, uint width, uint height, uint value )
+        {
+            const size_t x = get_global_id(0);
+            const size_t y = get_global_id(1);
+
+            if ( x < width && y < height ) {
+                in[offset + y * rowSize + x] = value
```

Comment:
There is no `in` as input parameter. Did you mean `data`?

Reply:
noted on this, forgot to change this while I did a change in params.